Dockerrun.aws.json

```diff
@@ -1,8 +1,7 @@
 {
   "AWSEBDockerrunVersion": 1,
   "Image": {
-    // TODO Update this with the appropriate prefix, and remove this comment
-    "Name": "woeip:latest",
+    "Name": "openoakland/woeip:latest",
```

Comment:
Docker images should be pulled from https://hub.docker.com/r/openoakland/woeip/. Let me know if you want access.